x-pack/plugins/security/server/authentication/providers/base.mock.ts

```diff
@@ -7,12 +7,20 @@
 import sinon from 'sinon';
 import { ScopedClusterClient } from '../../../../../../src/core/server';
 import { Tokens } from '../tokens';
-import { loggingServiceMock, httpServiceMock } from '../../../../../../src/core/server/mocks';
+import {
+  loggingServiceMock,
+  httpServiceMock,
+  elasticsearchServiceMock,
+} from '../../../../../../src/core/server/mocks';
 
 export type MockAuthenticationProviderOptions = ReturnType<
   typeof mockAuthenticationProviderOptions
 >;
 
+export type MockAuthenticationProviderOptionsWithJest = ReturnType<
```

Comment:
__note:__ want to migrate to a pure Jest mocks for a long time already (not that I like Jest mocks, they are limited comparing to sinon, but just for the consistency sake). So decided to write tests for PKI auth provider without sinon and then will be migrating provider tests one by one when I have a spare time.